app/views/pagy/_previous_page.html.erb

```diff
@@ -1,5 +1,5 @@
 <li class="pub-c-pagination__item pub-c-pagination__item--previous">
-  <%= link_to url, rel: 'prev', remote: remote, class: "pub-c-pagination__link govuk-link" do %>
+  <%= link_to "#{request.path}?page=#{pagy.prev}", rel: 'prev', class: "pub-c-pagination__link govuk-link" do %>
```

Comment:
this is rudimentary but works for our use case. I tried a couple of other things but would need a fair bit of work to deal with all the cases